### ISSUE TITLE
Setup Role for Fluentd td-agent(s) - Log Parser and Aggregator 

### ISSUE THREAD

**Issue by JJediny** [GSA/data.gov#21]
https://github.com/JJediny/ansible-td-agent
- [x] Setup Nginx module and td-agent on tagged servers
- [x] Setup Apache module and td-agent on tagged servers
- [x] Setup Tomcat module and td-agent on tagged servers
- [ ] Setup OSSEC tail -f of alerts.json module and td-agent on all servers
- [x] Setup auditd module and td-agent on all servers

See Issue #20 for full scope

**Comment by neilhunt1** [comment]
@JJediny do we want each of these on separate machines?

**Comment by neilhunt1** [comment]
@JJediny assigning this and #20 to me. Dan said these were priorities

**Comment by neilhunt1** [comment]
[This](https://github.com/GSA/ansible-td-agent) is forked repository in GSA space that we will be working from.

**Comment by neilhunt1** [comment]
The original ansible td-agent we cloned from continues to be problematic with Ubuntu (at least). So I have been trying out other ways to install td-agent through ansible and I found [this one](https://github.com/kawasakitoshiya/ansible-td-agent). With some slight modifications I have it working with our hardened Ubuntu image. Let me know if you have any issues with this, otherwise I will clone this one tomorrow and start integrating it.
